### ISSUE TITLE
Tab Completion

### ISSUE THREAD

**Issue by mstathers** [kilobyte/kbtin#1]
I understand tab completion is currently disabled because tab is used elsewhere (input buffer switching). Does using tab completion (Tab) stop the buffer switching from working (Ctrl-Tab, Alt-Tab)? I'm wondering if it would be possible to have all of the coexist.

**Comment by kilobyte** [comment]
Yes, this sounds like a good idea.  Too bad, I'm rather short at tuits these days so it might take a while to implement this.  I haven't done any mudding in a while so naturally improving the MUD client is of rather low priority.  But if I'd find some time, moving buffer switching to Ctrl-Tab and implementing tab completion sounds good.

I guess it would be good to include any recently seen text in the completion, too.

**Comment by mstathers** [comment]
> I guess it would be good to include any recently seen text in the completion, too.

Certainly! I think this is how most MUD client's do it anyway. Helps to with long item/people/place names. eg.
> Doctor Beezlebugheimershmitt enters the room
  look beezle<tab>
